### PR TITLE
Implement tests for various data adding handlers

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer_grpc/handlers/add_landmark_data_handler_test.cc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+namespace cartographer_grpc {
+namespace handlers {
+namespace {
+
+using ::testing::_;
+using ::testing::Eq;
+using ::testing::Pointee;
+using ::testing::Truly;
+
+const std::string kMessage = R"PROTO(
```
Comment (pifon2a) [inline cartographer_grpc/handlers/add_landmark_data_handler_test.cc]: in google3 I think we would have "constexpr char kMessage[] = ", but we are not in google3 :)

Comment (cschuet) [reply cartographer_grpc/handlers/add_landmark_data_handler_test.cc]: :)

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**